modules/ipi-install-preparing-the-bare-metal-node.adoc

```diff
@@ -53,52 +53,68 @@ $ vim bmh.yaml
+  name: openshift-worker-<num>-bmc-secret <2> 
+  namespace: openshift-machine-api
 type: Opaque
 data:
-  username: <base64-of-uid> <2>
-  password: <base64-of-pwd> <3>
+  username: <base64-of-uid> <3>
+  password: <base64-of-pwd> <4>
 ---
 apiVersion: metal3.io/v1alpha1
 kind: BareMetalHost
 metadata:
-  name: openshift-worker-<num> <1>
+  name: openshift-worker-<num> <2>
+  namespace: openshift-machine-api
 spec:
   online: true
-  bootMACAddress: <NIC1-mac-address> <4>
+  bootMACAddress: <NIC1-mac-address> <5>
   bmc:
-    address: <protocol>://<bmc-ip> <5>
-    credentialsName: openshift-worker-<num>-bmc-secret <1>
-  networkConfig: <6>
-    interfaces:
-    - name: <NIC1_name>
-      type: ethernet
-      state: up
-      ipv4:
-        address:
-        - ip: "<IP_address>"
-          prefix-length: 24
-        enabled: true
-    dns-resolver:
-      config:
-        server:
-        - <DNS_IP_address>
-    routes:
-      config:
-      - destination: 0.0.0.0/0
-        next-hop-address: <IP_address>
-        next-hop-interface: <NIC1_name>
-----
-<1> Replace `<num>` for the worker number of the bare metal node in the two `name` fields and the `credentialsName` field.
-<2> Replace `<base64-of-uid>` with the `base64` string of the user name.
-<3> Replace `<base64-of-pwd>` with the `base64` string of the password.
-<4> Replace `<NIC1-mac-address>` with the MAC address of the bare metal node's first NIC. See the BMC addressing section for additional BMC configuration options. Replace `<protocol>` with the BMC protocol, such as IPMI, RedFish, or others.
-<5> Replace `<bmc-ip>` with the IP address of the bare metal node's baseboard management controller.
-<6> Optional. You can set the `networkConfig` configuration option to configure host network interfaces. See "(Optional) Configuring host network interfaces in the `install-config.yaml` file" in the "Setting up the environment for an OpenShift installation" section for configuration details. 
+    address: <protocol>://<bmc-ip> <6>
+    credentialsName: openshift-worker-<num>-bmc-secret <2>
+    disableCertificateVerification: true <7>
+    username: <bmc-username> <8>
+    password: <bmc-password> <9>
+  preprovisioningNetworkDataName: openshift-worker-<num>-network-config-secret <10> <2>
+----
+<1> (Optional) To configure network for a newly created node, specify the name of the secret that contains the network configuration. Follow the `nmstate` syntax to define the network configuration for your node.
+<2> Replace `<num>` for the worker number of the bare metal node in the `name` fields, `credentialsName` field, and the `preprovisioningNetworkDataName` field.
+<3> Replace `<base64-of-uid>` with the `base64` string of the user name.
+<4> Replace `<base64-of-pwd>` with the `base64` string of the password.
+<5> Replace `<NIC1-mac-address>` with the MAC address of the bare metal node's first NIC. See the BMC addressing section for additional BMC configuration options. 
```

Comment:
@michaelryanpeter Good point. Instead of updating in the call out, I defined the acronym at it's first usage [here](https://github.com/openshift/openshift-docs/pull/45918/files#diff-f90b46cde12693fcb811ca202b3759cea7652922c2b7f92da22716e5e8aaa180R33).